orienteer-module-loader/pom.xml

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I do recommend to merge orienteer-module-loader to orienteer-core. After polishing it will become "micro-kernel" of Orienteer - so it will look strange to have micro-kernel as module for "something":)